cmd/kpod/create_cli.go

```diff
@@ -40,6 +42,81 @@ func addWarning(warnings []string, msg string) []string {
 	return append(warnings, msg)
 }
 
+func parseVolumes(volumes []string) error {
+	for _, volume := range volumes {
+		arr := strings.Split(volume, ":")
+		switch len(arr) {
```

Comment:
Should check for len(arr) < 2 and fail.  Should use Splitn(volume,":", 3)

Reply:
fixed